README.md

```diff
@@ -65,7 +65,30 @@ Depending on the context, ruby-build does a little bit more than the above: for
 
 #### Custom Build Definitions
 
-To install a version of Ruby that is not recognized by ruby-build, you can specify the path to a custom build definition file in place of a Ruby version number.
+To install a version of Ruby that is not available in ruby-build, you can specify the path to a custom build definition file in place of a Ruby version number.
+
+```sh
+# As a standalone program
+$ ruby-build -d path/to/3.4-custom /opt/rubies  # installs to /opt/rubies/3.4-custom
```

Comment:
```suggestion
$ ruby-build -d /path/to/3.4-custom /opt/rubies  # installs to /opt/rubies/3.4-custom
```

Reply:
`/` because the command should work regardless of the current working directory, and `/path/to/..` is I think also clearer it needs to be changed by the user.